### DISCUSSION TITLE
Synthetic document generator

### DISCUSSION THREAD

**Discussion by deClot** [mindee/doctr#1230] [General]
Hi everybody,
I'm retrain DBNet with my own document data obtained from FineReader or CRAFT but these methods are not working well on some complicate or noise documents. So I thought about synthetic document generation. 
What do you use as your dataset? Real documents or some document generator? Could  you give advise about how to get clean data to my dataset?

**Comment by felixT2K** [comment]
Hi @deClot 👋🏼 ,

The detection models are trained on real data (custom mindee dataset - not public available). @odulcy-mindee how many samples are inside the detection dataset ? :)
Does the already trained models not work well on your use case ?

A possible synthetic approach:
https://github.com/clovaai/synthtiger
with Azure: https://github.com/microsoft/genalog

**Reply by deClot** [reply]
> custom mindee dataset - not public available

Can you share how you got annotation on your documents? Used other engine as baseline or something else?

> Does the already trained models not work well on your use case ?

Yeah, some words (or their part) in beginning or end of lines are lost. I tried to increase input image size to detect its but it doesn't help.
![screenshot-2023-07-11_12-07-04](https://github.com/mindee/doctr/assets/11291800/7e6ca400-56e3-42ad-a0fb-b70b5fcb9837)

About genalog: I also tried this repo for document generation and now are figuring out how to create my own templates.
synthtiger: thanks, will try :)
